### PR TITLE
Fixed a few wholes in the English.

### PR DESCRIPTION
**Pull request by DannyB5544**
Gotta love lingo, lol

### CODE REVIEW

File: bettermod/bettermod.py [hunk 1]
```diff
@@ -730,7 +730,8 @@ def __init__(self, bot):
             await self.bot.kick(user)
         except:
             await self.bot.say("I cannot kick this user, he higher than me in the role hierarchy. Aborting...")
-            await self.bot.send_message(channel, content="The user was not kick. Check my permissions", embed=modlog)
+            await self.bot.send_message(channel, content="The user was not kicked. Check my permissions and verify I can kick users.", embed=modlog)
+", embed=modlogmodlo
```
Comment (retke) [inline bettermod/bettermod.py]: ok but why that line ?

Comment (DannyB5544) [reply bettermod/bettermod.py]: Er I'm not actually sure where 734 came from, I don't see it in my editor...

**Comment by DannyB5544** [comment]
That any better

**Comment by retke** [comment]
this should be fixed now

**Pull request merged**